airflow/www/extensions/init_appbuilder.py

```diff
@@ -211,7 +211,6 @@ def init_app(self, app, session):
         self.sm = self.security_manager_class(self)
         self.bm = BabelManager(self)
         self.openapi_manager = OpenApiManager(self)
-        self.menuapi_manager = MenuApiManager(self)
```

Comment:
I think we should remove the openapi_manager. We have an open API doc for our REST API. It seems that the MenuApiManager is a different thing and shouldn't be removed?

Reply:
I'm good with removing `OpenApi` too, even though that didn't actually add any endpoints in practice 🤷‍♂️.

We do want to remove `MenuApi`, which exposes `/api/v1/menu`. It's a FAB default we don't want, particularly now that we are removing the default FAB REST API auth endpoints too.